todo_app/app.py

```diff
@@ -11,132 +11,184 @@
 import pymongo
 from bson.objectid import ObjectId
 from datetime import datetime
+from flask_login import LoginManager
+from oauthlib.oauth2 import WebApplicationClient
+from flask_login import login_required
+from urllib import parse
+from flask import request
+from flask_login import UserMixin, login_user, current_user
+from todo_app.data.UserClass import User
+import json
+import logging
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+from logging import getLogger
 
 
 def create_app():
     app = Flask(__name__)
-
-    toDoId = os.environ['TOID']
-    doingId = os.environ['DOINGID']
-    doneId = os.environ['DONE']
-    dbconnect = os.environ['CLIENT']
+    githubId = os.environ['GITHUBID']
+    webClient = os.environ['WEBAPPLICATIONCLIENT']
+    clientSecret = os.environ['CLIENTSECRET']
+    todoBoard = os.environ['TODOBOARD']
+    doingBoard = os.environ['DOINGBOARD']
+    doneBoard = os.environ['DONEBOARD']
+    connectString = os.environ['CONNECTIONSTRING']
+    app.secret_key = os.getenv('SECRET_KEY')
+    app.config['LOGIN_DISABLED'] = os.getenv('LOGIN_DISABLED') == 'True'
+    app.logger.setLevel(os.getenv('LOG_LEVEL', 'INFO'))
+    # app.logger.setLevel(os.getenv('LOGGLY_TOKEN'))
+    if os.getenv('LOGGLY_TOKEN'):
+        handler = HTTPSHandler(
+            f'https://logs-01.loggly.com/inputs/{os.getenv("LOGGLY_TOKEN")}/tag/todo-app')
+        handler.setFormatter(
+            Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+        app.logger.addHandler(handler)
+    getLogger('werkzeug').addHandler(HTTPSHandler(
+        f'https://logs-01.loggly.com/inputs/{os.getenv("LOGGLY_TOKEN")}/tag/todoapp-requests'))
 
     def connectDb():
+
         client = pymongo.MongoClient(
-            dbconnect, ssl=True, ssl_cert_reqs='CERT_NONE')
-        print(client.list_database_names())
+            connectString, ssl=True, ssl_cert_reqs='CERT_NONE')
         database = client["01"]
         trello_collection = database["trello_collection"]
         todo = trello_collection.find(
-            {'idBoard': '6005828032dafa5707bf5dc3'}, {"name": 1})
+            {'idBoard': todoBoard}, {"name": 1})
         for x in todo:
-            print(x)
-            x
+            app.logger.info("Value of  todo board is %s", x)
```

Comment:
This will print out every item. That might be useful for debugging, but probably not for normal operation, so you could use the `debug` log level.

Reply:
Updated, thank you